spec/functional/resource/windows_env_spec.rb

```diff
@@ -0,0 +1,285 @@
+#
```

Comment:
I recommend doing a `git mv` here and then make the appropriate changes.

Reply:
I have tried by `git mv ` but still, in git file changes it shows file has been deleted you can see commit at https://github.com/chef/chef/pull/6612/commits/85ad3ee585ab50d173c083325d106967d2161dab#diff-5903ffd7bafff40eae05e9c21380c6a0